client-rpc/src/rpc/wallet_rpc.rs

```diff
@@ -80,6 +82,25 @@ pub trait WalletRpc: Send + Sync {
         view_keys: Vec<String>,
     ) -> Result<String>;
 
+    #[rpc(name = "wallet_buildRawTransferTransaction")]
+    fn build_raw_transfer_tx(
+        &self,
+        request: WalletRequest,
+        to_address: String,
+        amount: Coin,
+        view_keys: Vec<String>,
+    ) -> Result<String>;
+
+    #[rpc(name = "wallet_signRawTransferTransaction")]
+    fn sign_raw_transfer_tx(&self, request: WalletRequest, unsigned_tx: String) -> Result<String>;
+
+    #[rpc(name = "wallet_broadcastSignedTransferTransaction")]
+    fn broadcast_signed_transfer_tx(
```

Comment:
Is the result of `build_raw_transfer_tx` can be sent to tendermint broadcast directly? I think no need this api in `client-rpc`?

Reply:
no, I'll remove it, just for test

Reply:
It does an extra step of storing / tracking the tx before broadcast: https://github.com/crypto-com/chain/pull/1072/files#diff-56d5986577a1416a78d11832b08af3c0R865 but not sure if necessary?